generators/generator-bot-core-assistant/README.md

```diff
@@ -1,48 +1,34 @@
 # @microsoft/generator-bot-core-assistant [![NPM version][npm-image]][npm-url]
 
-This template creates an assistant-style conversational bot. Assistant-style bots typically help their users accomplish multiple different tasks, and have support for more varied conversational interactions.
+A bot with Azure Language Understanding (LUIS) and common trigger phrases used to direct the conversation flow and help customers accomplish basic tasks.
 
-Includes support for:
+### Recommended use
+
+- Create a sophisticated conversational bot
+- Customize and extend sophisticated dialogs, bot logic, language understanding and bot responses
+
+### Included capabilities
 
 - Greeting new and returning users
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
 - Asking for help
-- Cancelling a dialog
-- Submitting feedback about the bot
 - Error handling in conversations
+- Cancelling a dialog
+- Get customer feedback
 - Repeat the previous question
-- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
 - Disambiguation of NLP results
 
-## What this template is for
-
-Use this template if you want to...
-
-- Create an assistant-style, or advanced conversational bot
-- See examples of more complex conversational flows, and more advanced language understanding and generation
-
-## Languages
+### Required Azure resources
 
-- English (en-US)
-
-## Azure Resource Deployment
-
-To run this bot you'll need the resources listed below. Create a publishing profile in Composer to provision and publish to your Azure resources for your bot.
-
-- [LUIS][luis], or another recognizer of your choice
-- [QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
+- [Azure Language Understanding (LUIS)][luis], or another recognizer of your choice
+- [Azure QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
```

Comment:
Aren't Azure Bot Service, and (Web App or Functions) also required Azure resources? Or do we just say since that's a given so we don't list it. 

Or is it required just in the sense that it's what it takes to run it locally?

Reply:
It's a given.  we don't call out the execution environments or the table lookup on the service registration.  we do those automatically from within composer.  specifically for the execution env choice, you have to decide this up front when you're creating a new bot.